modules/react/modal/stories/examples/BodyOverflow.tsx

```diff
@@ -1,52 +1,52 @@
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
-            <p>Are you sure you want to delete the item?</p>
+            <p>
+              Permission is hereby granted, free of charge, to any person obtaining a copy of this
+              software and associated documentation files (the "Software"), to deal in the Software
+              without restriction, including without limitation the rights to use, copy, modify,
+              merge, publish, distribute, sublicense, and/or sell copies of the Software, and to
+              permit persons to whom the Software is furnished to do so, subject to the following
+              conditions:
+            </p>
+            <p>
+              The above copyright notice and this permission notice shall be included in all copies
+              or substantial portions of the Software.
+            </p>
+            <p>
+              THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+              INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+              PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+              HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF
+              CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE
+              OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+            </p>
           </Modal.Body>
           <HStack spacing="s" padding="xxs" marginTop="xxs">
-            <Modal.CloseButton as={DeleteButton} onClick={handleDelete}>
-              Delete
+            <Modal.CloseButton as={PrimaryButton} onClick={handlePrimaryClick}>
+              Primary Action
             </Modal.CloseButton>
-            <Modal.CloseButton>Cancel</Modal.CloseButton>
+            <Modal.CloseButton onClick={handleSecondaryClick}>Secondary Action</Modal.CloseButton>
```

Comment:
Same feedback here as I had for the Basic example: what do you think about replacing `Primary Action` and `Secondary Action` with less generic actions?